syntax = "proto3";

package serialize;

import "graph.proto";

message RoutingSettings {
    int32 bus_wait_time = 1;
    double bus_velocity = 2;
}

message StopId {
    string name = 1;
    uint64 id = 2;
}

message Router {
    RoutingSettings routing_settings = 1;
    repeated StopId stop_id = 2;
    Graph graph = 3;
}